### ISSUE TITLE
Unbinding events with off throws error

### ISSUE THREAD

**Issue by thasmo** [kenwheeler/cash#143]
When calling `element.off('click');` if `element.on('click', ...);` has not been called before, i.e. when the event is not set up yet, I get the error `TypeError: getData(...) is undefined`, seem to occur [here](https://github.com/kenwheeler/cash/blob/9f337ecba6503871c487af7e4627060f1e80e2af/src/events.js#L9).

**Comment by shshaw** [comment]
Ah, I think I see the issue. We're expecting an object and immediately trying to fetch a property of that object, but instead are receiving undefined. Good catch.

**Comment by shshaw** [comment]
Pushed a fix for this over at https://github.com/kenwheeler/cash/tree/event-off-fix

Try it out and let me know if you experience any issues. Next bugfix update we'll roll it into the master repo.

**Comment by bmcminn** [comment]
@thasmo, I saw the same error in Firefox, but I was getting a slightly different error in Chrome and I'm thinking it has to do with the JS interpreter in V8.

```
Firefox:
TypeError: getData(...) is undefined                                            cash.js:484:22

Chrome: 
Uncaught TypeError: Cannot read property '.delegateEventsview8' of undefined    cash.js:484:50
```

Hitting this issue with Backbone's delegated event handling in my views.

-----

Seems there isn't test coverage yet against running `.off()` on a bound event that hasn't been initialized per @thasmo's scenario: https://github.com/kenwheeler/cash/blob/master/test/test.js#L300-L310

**Comment by shshaw** [comment]
Fixed in [1.3.5](https://github.com/kenwheeler/cash/releases/tag/1.3.5)
